helm/vitess/examples/minikube.yaml

```diff
@@ -0,0 +1,71 @@
+topology:
+  cells:
+    - name: "zone1"
+      etcd:
+        replicas: 1
+      vtctld:
+        replicas: 1
+      vtgate:
+        replicas: 1
+      mysqlProtocol:
+        enabled: true
+        authType: "none"
+      keyspaces:
+        - name: "messagedb"
+          schema: |-
+            CREATE TABLE messages (
+              page BIGINT(20) UNSIGNED,
+              time_created_ns BIGINT(20) UNSIGNED,
+              message VARCHAR(10000),
+              PRIMARY KEY (page, time_created_ns)
+            ) ENGINE=InnoDB
+          vschema: |-
+            {
+              "sharded": true,
+              "vindexes": {
+                "hash": {
+                  "type": "hash"
+                }
+              },
+              "tables": {
+                "messages": {
+                  "column_vindexes": [
+                    {
+                      "column": "page",
+                      "name": "hash"
+                    }
+                  ]
+                }
+              }
+            }
+          shards:
+            - name: "0"
+              tablets:
+                - type: "replica"
+                  vttablet:
+                    replicas: 2
+
+etcd:
+  replicas: 1
+  resources:
+
+vtctld:
+  serviceType: "NodePort"
+  resources:
```

Comment:
Does this work with empty resources? It looks really strange. I would either expect to see `resources: {}` or just set some lower resources than in the default chart.

Reply:
It still feels strange. You don't want to just set low resource usage? You should be able to spin up your cluster and see how much each pod/container is consuming.

Reply:
Waiting for @dkhenry to weigh in. He's the one who recommended null resources.

In terms of resource, the concern I have is that these pods burn a lot of CPU when they come up, but then become mostly idle. Later, their load would go up depending on what the users would try. In other words, I don't know how to size these pods.

Reply:
CPU is less important since it just causes things to run slowly. It more the RAM constraints that I think are valuable, especially for someone wanting to know how big their VM needs to be for this example to come up.

I'm not against leaving null resources, but if we leave them null, it'd be helpful to have comments as to why they are null.

Reply:
@sougou For the example you can set Requests so people have an idea on how to size their VM
```
      requests:
        memory: "256Mi"
        cpu: "250m"
``

That would be for the tablets and gateways, and for the control containers you can cut down the ram a bit